docs/2-user-guide.md

```diff
@@ -449,15 +449,45 @@ For example:
 When the **member list** is **private**, other users will not be able to view the full member list of your system, but they can still query individual members given their 5-letter ID. If **current fronter** is private, but **front history** isn't, someone can still see the current fronter by looking at the history (this combination doesn't make much sense).
 
 ### Member privacy
-There is also an option to mark a specific member as private, using the command:
+<!-- There is also an option to mark a specific member as private, using the command:
```

Comment:
Are you sure comments like these work? They might still get rendered by Jekyll (if you tested this, nevermind)

Reply:
comments like that do get rendered as in they are put into the Dom, but they are put into the DOM as is. which means they are still invisible to the end user as the comment is an html comment. however that should have been removed anyway and I'll go remove it when fixing all the reviews